### DISCUSSION TITLE
Arctic separate coroutine

### DISCUSSION THREAD

**Discussion by joshuaspear** [bmoscon/cryptofeed#751] [Q&A] [answered]
Hi, I just wondered whether there was a specific reason that the arctic backend is not assigned it's own coroutine - similar to how the Postgres backend is setup? (Apologies, if I have miss understood - I am relatively new to async). I would be happy to implement the functionality via a PR if not. 

 Thanks, 

Josh

**Comment by bmoscon** [comment, marked as answer]
arctic doesnt support asyncio, so there isnt really any point

**Reply by joshuaspear** [reply]
Thanks for confirming
